Fixed issues #6/7/8/11 - Isaac Staugaard

isaacstaugaard opened the pull request:
fixes issue #6
fixes issue #7 
fixes issue #8 
fixes issue #11

PazZaitGivon commented:
cant find functionality mentioned in 11; and readme is still wrong. You are missing about 50 points to 500. If you fix it by tommorrow and submit a new pull request Ill take a look

isaacstaugaard commented:
Ohhhhh.  I forgot to push the build.xml file which contained the new ant
Run target. The 'ant Run' command runs the combined translator. I fixed the
PigLatin translator to just convert PigLatin. Everything should work now, I
updated the README.md screenshot to reflect the new targets. I think
everything should work now, including issue 6 and 11. I just pulled.
Thanks!
Isaac Staugaard

On Sat, Mar 14, 2015 at 1:39 PM, Paz Zait-Givon <notifications@github.com>
wrote:

> cant find functionality mentioned in 11; and readme is still wrong. You
> are missing about 50 points to 500. If you fix it by tommorrow and submit a
> new pull request Ill take a look
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/UCSB-CS56-Projects/cs56-misc-translate-to-secret-languages/pull/10#issuecomment-80712842>
> .
>
